README.md

```diff
@@ -106,7 +111,9 @@ The interaction with OpenShift is via the command line tool _oc_ which is copied
 <a name="starting-minishift"></a>
 ### Starting Minishift
 
-1. Assuming you have put _minishift_  on the _PATH_ as described in [Installing Minishift](#installing-minishift) you can start Minishift via:
+1. Run the `minishift start` command. The following example shows the result of
+the command when you start Minishift on a Linux operating system with the KVM
```

Comment:
why do we use Linux as an example here? Why not windows or OS X? That are out bigger target groups.